oauth2client/client.py

```diff
@@ -134,6 +134,13 @@ class AccessTokenRefreshError(Error):
     """Error trying to refresh an expired access token."""
 
 
+class HttpAccessTokenRefreshError(AccessTokenRefreshError):
+    """Error (with HTTP status) trying to refresh an expired access token."""
+    def __init__(self, *args, **kwargs):
+        super(HttpAccessTokenRefreshError, self).__init__(*args)
```

Comment:
I'm not aware that the base exception class does anything with keyword arguments to begin with, so I don't think forwarding them to the base class is a problem.  If you disagree, I'm find with including them.